docs/blog/100days/serverless/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Challenge 6 - Use Serverless Functions and Services to Collect Form Data
+date: 2020-02-04
+author: "Hashim Warren"
+excerpt: "Learn how to use a new class of services that pair well with your Gatsby site"
+tags: ["learning-to-code", "contest", "100-Days-of-Gatsby"]
+---
+
+_Gatsby was named the [#1 new technology to learn in 2020](https://www.cnbc.com/2019/12/02/10-hottest-tech-skills-that-could-pay-off-most-in-2020-says-new-report.html)!_
+
+_To help you learn about Gatsby and our ecosystem, we invite you to take the #100DaysOfGatsby challenge! If you are new to Gatsby, you can follow along with our beginner-friendly weekly prompts. Or if you are familiar with Gatsby and want to go deeper, we invite you to [build your first Gatsby Theme](/docs/themes/building-themes/)._
+
+_Learn more about [#100DaysOfGatsby here](/blog/100days)!_
+
+## Challenge 6: Collect Form Data Without Managing Your Own Database or Server
+
+In the last two challenges you added forms to Markdown and JSX-powered pages. This week we’d like you to create a home for your form data, using a new class of services that work well with [serverless websites](/blog/2018-10-10-unbundling-of-the-cms#integrating-modular-services), like your Gatsby project.
+
+Below we’ve provided resources for two paths that you can choose based on the needs for your projects, and your own interest and abilities.
+
+_Path A_ introduces you to form services for static sites, like Formik (the cloud service), Statickit, and FormKeep. Instead of giving you an all-in-one form builder to work with, these powerful, flexible, and inexpensive services provide you with endpoints to send form data to.
+
+_Path B_ welcomes you to the world of functions-as-a-service from the major cloud platforms. By using a little code, and a lot of configuration, you can create your own data workflow with one of the major cloud providers, like AWS, Google, and Azure. There are two benefits to this path. First, instead of using yet-another-SaaS, you’ll enhance your Gatsby site with the cloud provider services your employer has already invested in. Or, if you’re not familiar with the serverless tools provided by AWS, Google, and Azure, this challenge can be your first introduction to them.
```

Comment:
```suggestion
_Path B_ welcomes you to the world of functions-as-a-service from the major cloud platforms. By using a combination of code and configuration, you can create your own data workflow with providers like AWS, Google, and Azure. There are two benefits to this approach. First, instead of using yet-another-SaaS, you’ll enhance your Gatsby site with the cloud provider services your employer has already invested in. Second, if you’re not familiar with the serverless tools provided by AWS, Google, and Azure, this challenge can be your first introduction to them.
```
Removed the "a lot of configuration" just so as not to discourage people.

Reply:
Good point! Thank you for having learner empathy